buildpacks/nodejs-engine/buildpack.toml

```diff
@@ -10,9 +10,6 @@ keywords = ["node", "node.js", "nodejs", "javascript", "js"]
 [[buildpack.licenses]]
 type = "MIT"
 
-[[stacks]]
-id = "heroku-18"
```

Comment:
The CI matrix entry can be removed too:
https://github.com/heroku/buildpacks-nodejs/blob/153256103e0e1606e475b66480e0b6236761f0ec/.github/workflows/ci.yml#L81

Reply:
Thanks. Fixed in [1eaa234](https://github.com/heroku/buildpacks-nodejs/pull/528/commits/1eaa234a4b6dbf69876df37da0b01dbd70fa8e5e).